tools/legacy/benchmark_app/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 set(TARGET_NAME benchmark_app_legacy)
 
 file (GLOB SRC ${CMAKE_CURRENT_SOURCE_DIR}/*.cpp)
-file (GLOB HDR ${CMAKE_CURRENT_SOURCE_DIR}/*.hpp)
+file (GLOB HDR (${CMAKE_CURRENT_SOURCE_DIR}/*.hpp ${CMAKE_CURRENT_SOURCE_DIR}/*.h))
```

Comment:
can we just rename header to .hpp? It seems to contain C++ code

Reply:
That fiale was copied somewhere from common library during benchmark_app separation from samples code. So I don't want to rename it frankly

Reply:
still insists on a proper fix

Reply:
Again, this change is made because of file w_dirent.h copied from openvino\samples\cpp\common\utils\include\samples\os\windows\w_dirent.h
during separation of benchmark_app_legacy from openvino samples.
I can rename it to .hpp extension, but then we'll have to explain to much people why do we have 2 completely identical files with different extensions. If you still insist, I will rename it. @vladimir-dudnik, please advice.

Reply:
let's rename these copies in both places and we don't have to explain anything

Reply:
The idea was to have legacy BA with as little changes as possible. That means file names shouldn't be changes if not needed.

So far the only need to change file name is
> It seems to contain C++ code

Every file written in C++ contains C++ code: .h, .hpp, .cpp. The need doesn't seem to be strong enough to me. Legacy BA is not about consistency in code style.